andela/travel_tool_back
# #160710724 Enable Travel Administrator to View Further Guest House Details

#### What does this PR do?
- Creates an endpoint travels administrators to view further guest house details
- Enable admin to fetch timeline data for a room's bed trips

#### Description of Task to be completed?
- Fetch further guest house details: be able to get the number of rooms, vacant spaces
#### How should this be manually tested?
- Clone the repository and enter the project's folder
- Checkout to this branch and pull the branch
- Assign yourself the travel admin role by changing your roleId to 29187 in the users table
- Start the application using 'make start'
- Open postman and provide your token in the authorization header. Enter url `GET 127.0.0.1:5000/api/v1/guesthouses/<guesthouse-id>`
 - If you do not have a guest house id, see [this](https://github.com/andela/travel_tool_back/pull/65) for guesthouse structure to create one
- You should get the full details of the guest house
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#160710724](https://www.pivotaltracker.com/story/show/160710724)
#### Screenshots (if appropriate)
<img width="731" alt="screen shot 2018-10-03 at 16 09 41" src="https://user-images.githubusercontent.com/23254512/46412646-cbb3a480-c727-11e8-8304-e1f89e12c670.png">

#### Questions:
